game/scripts/vscripts/components/points/points.lua

```diff
@@ -1,12 +1,19 @@
-
 -- Taken from bb template
 if PointsManager == nil then
-    DebugPrint ( '[points/points] creating new PointsManager object' )
-    PointsManager = class({})
+  DebugPrint ( '[points/PointsManager] Creating new PointsManager object.' )
```

Comment:
Leave the debug line to be the filename, since the intention is for programmers to find the file sending the messages.